recipes/cppheaderparser/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "CppHeaderParser" %}
+{% set version = "2.7.4" %}
+{% set sha256 = "382b30416d95b0a5e8502b214810dcac2a56432917e2651447d3abe253e3cc42" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+
+  run:
+    - python
+    - ply
+
+test:
+  imports:
+    - CppHeaderParser
+
+about:
+  home: http://senexcanis.com/open-source/cppheaderparser/
+  license: BSD
+  license_file: README.txt
```

Comment:
Please include a copy of the license file

Reply:
Sure. I didn't include the file because the original package only specifies "BSD" (on pip it's "BSD License") so I wasn't sure which specific BSD license this refers to. So I thought I'd refer to the original README.txt file. Which file should I add?